test/test_gdrcopy.py

```diff
@@ -32,6 +32,49 @@ def setUp(self):
         """Disable logging output messages"""
         logging.disable(logging.ERROR)
 
+    @ubuntu
+    @docker
+    def test_defaults_ubuntu(self):
+        """Default gdrcopy building block"""
+        g = gdrcopy()
+        self.assertEqual(str(g),
+r'''# GDRCOPY version 2.2
```

Comment:
The default version is still 2.1, right?  I don't think this test will pass as-is.

Reply:
Thanks @samcmill. Should I keep around all the old tests for 1.3 and 2.1? I notices some things are tested in 2.1 and not 1.3, and vice versa.

Reply:
The "default" tests are intended to call the building block with no options (i.e., just the defaults).  If you're going to bump the default version to 2.2 (probably a good idea), then these tests should reflect that.  In addition, add a new 2.1 specific test that sets `version='2.1'` to cover the 2.1 code branch.